system/Images/Handlers/ImageMagickHandler.php

```diff
@@ -72,7 +72,14 @@ public function _resize(bool $maintainRatio = false)
 		$source = ! empty($this->resource) ? $this->resource : $this->image->getPathname();
 		$destination = $this->getResourcePath();
 
-		$action = $maintainRatio === true ? ' -resize ' . $this->width . 'x' . $this->height . ' "' . $source . '" "' . $destination . '"' : ' -resize ' . $this->width . 'x' . $this->height . '\! "' . $source . '" "' . $destination . '"';
+		//todo FIX THIS HANDLER PROPERLY
```

Comment:
What's needed to fix this properly?

Reply:
Making sure it works on windows 7 and other OS. Windows (10/2012) doesn't like the \! For whatever reason. I just don't have access to other OS and not really enough time since I'm using this as our platform! :)